app/models/invenio_rdm_record_converter.rb

```diff
@@ -518,17 +520,36 @@ def funding(file_id)
     funding_sources.compact
   end
 
-  def map_collections_to_communities
-    @dh_collections.map do |collection_path|
-      prism_community_id_from_collection_path(collection_path)
+  def dh_collection_to_prism_community_collections
```

Comment:
Given your comment below:

```suggestion
  def dh_collection_to_prism_community_collection
```